Cargo.toml

```diff
@@ -40,6 +40,11 @@ walker = "^1.0.0"
 [target.x86_64-unknown-linux-gnu.dependencies.inotify]
 version = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```

Comment:
I've never published a crate, but that makes a lot of sense. Do you mind if I keep this PR open in the meantime?